src/Header.module.css

```diff
@@ -542,7 +542,7 @@ input[type=range]::-webkit-slider-thumb {
   background-color: #FFF;
   transform: translateX(400px);
   overflow: auto;
-  z-index: 1;
+  z-index: 3;
```

Comment:
Why do we need another crazy z-index hack?

Reply:
I agree, there is way too much z-indexing going on. Also, there is a lot of absolute positioning where a simple flexbox or grid would be a better choice. 

The cleanest way to fix it is to reorder the dom hierarchy. 

It's quite a few minor code changes to do it. I think it would be better to do them in a separate PR as it isn't specifically related to this fix.

Reply:
https://github.com/webaverse/app/pull/3085

Reply:
I closed #3085 because the fixes to the dom are no longer relevant on `z-targeting` and it'll just create a ton of conflicts. 

I'll do the z-index overhaul in `z-targeting`  as part of this issue #3087 

this PR merges into `z-targeting` just fine as is.